src/littlefs.c

```diff
@@ -0,0 +1,110 @@
+#include "littlefs.h"
+#include "w25x.h"
+
+#define BLOCK_SIZE 4096
```

Comment:
again - please avoid #defines if not necessary

Reply:
It would work in C++, but not possible in C (initializer element is not constant when setting the value to const struct)